source/common/upstream/health_checker_impl.h

```diff
@@ -280,6 +281,7 @@ class GrpcHealthCheckerImpl : public HealthCheckerImplBase {
     void decodeHeaders(Http::HeaderMapPtr&& headers, bool end_stream) override;
     void decodeData(Buffer::Instance&, bool end_stream) override;
     void decodeTrailers(Http::HeaderMapPtr&&) override;
+    void decodeMetadata(Http::MetadataMap&) override {}
```

Comment:
These ones we're probably never going to really use, but we could unit test the functions to avoid a coverage decrease

Reply:
Good idea! Test added.